helpers/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @description Clone an Object deeply nested.
+ * @param {any} object - Object to clone.
+ * @returns {any}
+ */
+function cloneObject(object: any): any {
```

Comment:
Lodash does not do that ? 
prefer a custom. mehtod ? @rimiti @Theo-Le-Donne

Reply:
https://codeburst.io/why-you-shouldnt-use-lodash-anymore-and-use-pure-javascript-instead-c397df51a66

Reply:
```
function cloneObject(object: any): any {
```

Why any if your parameter is named object ? 
You should never use "any"